src/pages/AdminEditUser.js

```diff
@@ -60,6 +69,7 @@ export const AdminEditUser = hh(class AdminEditUser extends Component {
       userId: userId,
       displayName: this.state.displayName,
       emailPreference: this.state.emailPreference,
+      institutionId: this.state.institutionId
     };
     const payload = {updatedUser: user};
```

Comment:
Currently when I update a user's institution, display name, etc, and hit save, changes are reverted once I refresh. I think the issue is wrapping the updated properties in `{updatedUser: user}` since the new endpoint expects json that can be converted directly to `UserUpdateFields`

Reply:
@shaemarks is correct here - the payload format is different for `/api/user/${userId}` than it is for `/api/dacuser/${userId}`. See [PUT /api/user/${userId}](https://consent.dsde-dev.broadinstitute.org/#/User/put_api_user__userId_) for what the payload should look like. Only values that are populated in the payload are updated, so anything not there will be ignored.

Reply:
For a little bit of history, the payload for `/api/dacuser/${userId}` used to take all sorts of arguments. We used to be able to swap one user for another for certain roles which was a huge headache. We've simplified it quite a bit on both sides, but the `{updatedUser: user}` structure is a holdover from that time.